public/components/overview/mitre_attack_intelligence/module_mitre_attack_intelligence_resource_flyout_subTables.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Wazuh app - React component for showing the Mitre Att&ck intelligence flyout tables.
+ *
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React , {useState, useEffect} from 'react';
+import { WzRequest } from '../../../react-services';
+import {
+  EuiBasicTableColumn,
+  EuiAccordion,
+  SortDirection,
+  EuiInMemoryTable,
+} from '@elastic/eui';
+import { Markdown } from '../../common/util/markdown';
+
+export const GetFlyoutSubtable = ({subtableName, subtableArray, subtableProps}) => {
```

Comment:
`Get` sounds like an action  its a component 
maybe `ReferencesTable`

Reply:
what means `subtableArray`? 
I think `references` is a more indicated name instead of `subtable`
`referenceName`
`referenceItems`
`tableProps`